### PR TITLE
Feature/settle 0

### PR DESCRIPTION
**Pull request by jijordre**
### Purpose
This PR enables the start of settlements with stage amount value of 0. This enables the settlement for wallets that have 0 active balance and still should settle for the staging of accrued.

### Changes
* Update `src/commands/settle.js` to support the start of settlement of amount of 0
* Move test on 0 amount out of `parseAmount()` and into `src/commands/unstage.js` and `src/commands/withdraw.js`.

### Issues
* #103 

### Checklist
_Please check if your PR fulfills the following requirements:_
- [x] Associated issue has been created and referenced above
- [x] Package version number has been updated according to semantic rules
- [x] Tests for the changes have been added and all test pass (`npm test`)
- [x] Test coverage requirements have been met or improved (`.nycrc`)
- [x] Eslint has been run and reported issues have been fixed (`npm run eslint`)
- [ ] Codeclimate has been run and reported issues have been fixed (`npm run codeclimate`)

### CODE REVIEW

**Review by hubiierik — CHANGES_REQUESTED** [review]
There seems to be a slight obsession about zero amount and at the same time accepting negative amount.

File: src/commands/unstage.js [hunk 1]
```diff
@@ -35,6 +35,9 @@ module.exports = {
             const tokenInfo = await provider.getTokenInfo(argv.currency);
             
             const amount = utils.parseAmount(argv.amount, tokenInfo.decimals);
+            if (amount.eq(0))
```
Comment (hubiierik) [inline src/commands/unstage.js]: Would it be reasonable to reject negative values?

File: src/utils.js [hunk 2]
```diff
@@ -13,9 +13,6 @@ function parseAmount(amount, decimals) {
         throw new TypeError('Amount must be a number!');
     }
 
-    if (amountBN.eq(0))
```
Comment (hubiierik) [inline src/utils.js]: Would it be reasonable to reject negative values?

File: src/utils.js [hunk 3]
```diff
@@ -13,9 +13,6 @@ function parseAmount(amount, decimals) {
...
-        throw new Error('Amount must be greater than zero!');
```
Comment (hubiierik) [inline src/utils.js]: Would it be reasonable to reject negative values?

**Review by hubiierik — CHANGES_REQUESTED** [review]

File: src/commands/unstage.js (hunk 1 above)
Comment (jijordre) [reply src/commands/unstage.js]: Good catch. Will update accordingly.

**Review by hubiierik — APPROVED** [review]

**Pull request merged**